src/components/common/DynamicTabs/useTabs.ts

```diff
@@ -138,8 +138,7 @@ export function useTabs(persistanceKey: string) {
                     isSelected: false,
                     /* NOTE: following lines migrate old tab data to new */
                     lastSyncMoment: dayjs(),
-                    // @ts-expect-error relic of old type
-                    position: _tab.positionFixed ? index : Number.MAX_SAFE_INTEGER,
+                    position: 'positionFixed' in _tab && _tab.positionFixed ? index : Number.MAX_SAFE_INTEGER,
```

Comment:
Don't the two checks do the same thing? `_tab.positionFixed` will return undefined if the property is not found and similarly `'positionFixed' in _tab` also does the same thing?

Reply:
No they are different. It is done to cater the typing issue here